tests/unit/behaviors/TranslatableModelTest.php

```diff
@@ -101,4 +102,52 @@ public function testSetTranslationValue()
         $this->assertEquals(['a', 'b', 'c'], $obj->states);
     }
 
+    public function testTranslateWhere()
+    {
+        $this->recycleSampleData();
+
+        $obj = CountryModel::first();
+
+        $obj->translateContext('fr');
+        $obj->name = 'Australie';
+        $obj->save();
+
+        $this->assertEquals(0, CountryModel::transWhere('name', 'Australie')->count());
+
+        Translator::instance()->setLocale('fr');
+        $this->assertEquals(1, CountryModel::transWhere('name', 'Australi')->count());
+
```

Comment:
Is that a typo, should it be "Australie" instead of "Australi" ?

Reply:
Indeed^^